packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -11,6 +11,7 @@ import { UPlotConfigBuilder } from '../config/UPlotConfigBuilder';
 import { CloseButton } from './CloseButton';
 
 export const DEFAULT_TOOLTIP_WIDTH = 280;
+export const DEFAULT_TOOLTIP_HEIGHT = 280;
```

Comment:
What's the data-viz convention for using `enum`s here vs `const`s? I've seen both in the code base, but would love to understand if our squad has a de facto convention.

Reply:
Not sure that I understand the question. `const` is something to avoid reassigning and `enum` is a type. I don't see any `versus` situation here, you even can use `const enum` if needed 😁
I'd like to use `const` when it's just a simple type variable like `number`, `string`, etc. and `enum` for a set of constants.

There is no specific convention in the team, just keep in mind that your code gonna read someone else, so keep it clean and readable/maintainable as possible 😁

Reply:
What I mean was, in my experience in the grafana code base, enums are often used when defining a collection of related constants - which these two values seem to be. I've seen other developers go the enum path when choosing to define related values in this way, so I was just curious if we had a convention. 😁👍

Reply:
Hm, good point! I don't have a strong opinion about it, I'm trying to use common sense to organize constants... We can make some guideline 😁 cc @nmarrs 
Looks like in this particular case we can use enum.

Reply:
I really don't have any strong feelings either way, I just want to make sure I'm adhering to the principle of making all of our visualizations as similar as possible. I know standardization is part of the plan moving forward, so I think having a solid understanding as a team of what we want our conventions to be would be helpful. :-)

Reply:
first rule of dataviz, there are no rules.

consts tend to be faster than enums (unless they're `const enum`s :)

Reply:
haha love this thread :) 

We don't have any hard and fast rules - i think enums mainly make sense in more conceptual cases (like [this one with canvas constraints](https://github.com/grafana/grafana/blob/50e43c48596525cfc275b1bd1cb2a3c629ce8b91/packages/grafana-schema/src/raw/composable/canvas/panelcfg/x/CanvasPanelCfg_types.gen.ts#L16)) - I guess case could be made here for an enum like tooltipDimensions or something but I don't feel too strongly on this